Google.PowerShell/ReleaseFiles/GoogleCloud.psd1

```diff
@@ -95,7 +95,8 @@ PrivateData = @{
     PSData = @{
 
         # Tags applied to this module. These help with module discovery in online galleries.
-        Tags = 'Google', 'Cloud', 'Compute', 'Storage', 'GCS', 'GCE', 'SQL', 'DNS'
+        Tags = 'Google', 'Cloud', 'Compute', 'Storage', 'GCS', 'GCE', 'SQL', 'DNS', 'PubSub', 
+               'Logging', 'PSEdition_Core', 'PSEdition_Desktop'

```

Comment:
Does this mean more modules are being added?

Reply:
This just means it we are saying that this module works on both PowerShell Core and normal PowerShell.